assets/src/edit-story/masks/frame.js

```diff
@@ -169,7 +172,11 @@ export default function WithMask({ element, fill, style, children, ...rest }) {
     >
       <svg width={0} height={0}>
         <defs>
-          <clipPath id={maskId} clipPathUnits="objectBoundingBox">
+          <clipPath
+            id={maskId}
+            transform={'scale(1 ' + heightMultiplier + ')'}
```

Comment:
Minor nit, might read a bit better:

```suggestion
            transform={`scale(1 ${heightMultiplier})`}
```